ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -101,3 +101,14 @@ spec:
         - mirrors:
           - registry.example.com:5000
           source: registry.redhat.io
+    #
+    # Application Namespaces should have Network Policies defined
+    #
+    - fileName: AcceleratorsNP.yaml
+      policyName: "config-policy"
+    - fileName: AcmAddonNP.yaml
+      policyName: "config-policy"
+    - fileName: AcmNP.yaml
+      policyName: "config-policy"
+    - fileName: AssistedInstallerNP.yaml
+      policyName: "config-policy"
```

Comment:
Shouldn't ACM/AssistedInstaller own the NetworkPolicies for these namespaces? They create them on-demand as needed. I don't think we should be attempting to manage their namespaces or policies.